### DISCUSSION TITLE
How to make flatpak apps visible

### DISCUSSION THREAD

**Discussion by jcv-dev** [prasanthrangan/hyprdots#1391] [Q&A] [answered]
Hey! It's my first time using Hyprdots and I'm loving it so far, but my flatpak apps don't show up when I use the app menu (Super + A). I was wondering if this is a problem on my end or expected behavior. On a side note, my flatpak apps can't open browser links, but that's less annoying. Thanks in advance!

**Comment by dieBakterie** [comment]
Regarding your Apps can't open Browser links you should take a look and Change this in `flatseal`, it's a flatpak for de-sandboxing flatpaks but why they aren't show up in the Rofi Menü i don't know.

**Comment by AKA-Akhil** [comment, marked as answer]
Hey even i am new to Hyperdots.Even i was facing a similar issue with Android Studio. Rofi relies on desktop entry files to display applications. Ensure that the desktop entry file is located in one of the directories where Rofi searches for applications. Typically, these directories include /usr/share/applications/ and ~/.local/share/applications/. If the desktop entry file is missing, you can create one.
